www/shared/utils/google-tag-manager/initialize-tag-manager.test.js

```diff
@@ -0,0 +1,31 @@
+const initializeMock = jest.fn();
+
+jest.doMock('react-gtm-module', () => ({ initialize: initializeMock }));
+
+beforeEach(() => {
+    process.env.GTM_TRACKING_ID = 'TESTE-123123';
+});
+
+afterEach(() => {
+    jest.resetModules();
+    jest.clearAllMocks();
+    jest.unmock('ganalytics');
+});
+
+it('should initialize gtm module if tracking id is provided', () => {
+    const initializeTagManager = require('./initialize-tag-manager').default;
```

Comment:
Move these requires into a single import at the start of the file.

Reply:
I have to require the file in the beginning of every test, but I moved it to a beforeEach and works just fine ☺️

Reply:
You have to re-require because `tagManagerInitialArgs` does not change between each test when you change `process.env`, right? One way to solve is to move `tagManagerInitialArgs` into the factory function. I'm 100% this is the actual issue, but I do have feeling this might be it.